site/content/en/latest/concepts/gateway-api-extensions.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Gateway API Extensions"
+---
+## Before you Begin
+- [The Gateway API](https://gateway-api.sigs.k8s.io/)
+
+## Overview
+
+Gateway Extensions are mechanisms that allow implementation-specific features to be added to the Kubernetes Gateway API without modifying its core specification. They enable Gateway API implementations like Envoy Gateway to expose their unique capabilities while maintaining compatibility with the standard API. Extensions follow defined patterns that preserve the role-oriented design of Gateway API while allowing for customization to address specific use cases not covered by the core API.
```

Comment:
should we have a dedicated page for Gateway API Extensions or add it under https://gateway.envoyproxy.io/docs/concepts/concepts_overview/#envoy-gateway-eg-api-resources
cc @missBerg ?